chdb/build_linux_arm64.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash -e
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+PROJ_DIR=$(dirname ${DIR})
+
+[ -z "${PYTHON_VERSIONS}" ] && { echo "please provide PYTHON_VERSIONS env, e.g: PYTHON_VERSIONS='3.8 3.9'"; exit 1; }
+
+# echo ${PROJ_DIR}
+
+for PY_VER in ${PYTHON_VERSIONS}; do
+    cd ${PROJ_DIR}
+    pyenv local "${PY_VER}"
+    python3 --version
+    python3 -m pip install pybind11
+    export CC=/usr/bin/clang
+    export CXX=/usr/bin/clang++
+    # Install universal2 pkg
+    python3 -VV
+    python3-config --includes
+    # if python3 -VV does not contain ${PY_VER}, then exit
+    if ! python3 -VV 2>&1 | grep -q "${PY_VER}"; then
+        echo "Error: Required version of Python (${PY_VER}) not found. Aborting."
+        exit 1
+    fi
+
+    python3 -m pip install -U pybind11 wheel build tox
+    rm -rf ./buildlib
+
+    ./chdb/build.sh
```

Comment:
I'm watching the runner build. Since the files inside the chdb build folder don't change, maybe this step could be moved ahead of the loop and completed once for all executions to speed up things?

Reply:
Python header files changed, so the `_chdb.python.xxx.so` will be different. It's necessary to run `./chdb/build.sh` for every python version.

Reply:
ACK. Will we need a better cleanup strategy to avoid filling up the builder storage.